### PR TITLE
Small fixes around action, updates

### PR DESCRIPTION
**Pull request by julianoes**
A bit of a mixed bag to get it working properly for me, see commits.

### CODE REVIEW

**Comment by unipheas** [comment]
Just checked, all works and looks good.

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ services:
             - "50051:50051"
             - "14540:14540/udp"
     gazebo_sitl_headless:
-        image: jonasvautherin/px4-gazebo-headless:v1.9.2
+        image: jonasvautherin/px4-gazebo-headless:v1.10.2
```
Comment (JonasVautherin) [inline docker/docker-compose.yml:16]: Does that work? Latest version is `1.10.1`

Comment (julianoes) [reply docker/docker-compose.yml:16]: There is a tag though: https://github.com/PX4/Firmware/releases/tag/v1.10.2
Strange...

Comment (JonasVautherin) [reply docker/docker-compose.yml:16]: Yes, but this has to be done manually in the px4-gazebo-headless repo :-). We just haven't done it, so the latest `px4-gazebo-headless` tag is `1.10.1` :see_no_evil:

Comment (unipheas) [reply docker/docker-compose.yml:16]: I didn’t test it, sorry

Comment (julianoes) [reply docker/docker-compose.yml:16]: Oh! Let me change it then.

Comment (julianoes) [reply docker/docker-compose.yml:16]: https://github.com/mavlink/MAVSDK-JavaScript/pull/13

**Pull request merged**